ironic_cleanup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function wait_condition () {
+condition=0
+while [ "$condition" != "3" ] ; do
+  condition=$( openstack baremetal node list -c 'Provisioning State' | grep $1 | wc -l)
```

Comment:
what is $1 in this case? maybe it's worth to use a local variable and assign $1 to it, for clarity